aip/client-libraries/4210.md

```diff
@@ -68,6 +146,12 @@ client library generator (in other words: the libraries that the generators
 write). Client libraries **must** implement these concepts in order to be
 considered complete.
 
+### Messages and Enums
+
+Client library generators **should not** generate code for `message` or `enum`
```

Comment:
Or service stubs which are already generated by gRPC?

Reply:
Hm you mean a like a grpc client stub for a `service` that would then be used by a GAPIC? If so, then yeah, I agree and will add.

Just want to clarify because of course w should be generating client types for `service`s 🙃

Reply:
I added a bullet to the existing `Services and methods` section below this to that effect, lmk.

Reply:
Looks good, thanks.